src/GraphQl/Resolver/ItemResolver.php

```diff
@@ -74,4 +74,25 @@ public function __invoke($source, $args, $context, ResolveInfo $info)
 
         return $this->normalizer->normalize($item, ItemNormalizer::FORMAT, $normalizationContext + $baseNormalizationContext);
     }
+
+    /**
+     * Recursively replaces the key "_id" (the raw id i GraphQL) by "id" (the name of the property expected by the Serializer).
+     */
+    private function replaceIdKey(array $array): array
+    {
+        foreach ($array as $key => $value) {
+            if ('_id' === $key) {
+                $array['id'] = true;
```

Comment:
Maybe using `$array['_id']` instead of `true` would be better in case the implementation in `ResolveInfo` changes.